content/en/docs/High Availability/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "High Availability"
+linkTitle: "High Availability"
+weight: 170
+description: >
+  Learn how to configure High Availability in Kyverno
+---
+
+## Configure Kyverno in HA mode
+To install Kyverno in HA using Helm Chart.
+  
+  **NOTE:** Due to some complexities accompanied with running 2 replicas, the recommended replica counts for HA is at least 3.
+
+```sh
+helm install kyverno kyverno/kyverno --create-namespace --set=replicaCount=3 
+```
+## How HA works in Kyverno
+This section provides details on how Kyverno handles HA scenarios.
+ **Module - Webhook Server**
```

Comment:
This heading isn't right. It should be a level three heading (three octothorpes preceding it).